src/status_im/integration_test.cljs

```diff
@@ -179,8 +179,8 @@
          (logout!)
          (rf-test/wait-for [::logout/logout-method]))))))
 
-(deftest delete-chat-test
-  (log/info "========= delete-chat-test ==================")
+(deftest close-chat-test
```

Comment:
could you pls elaborate on why we rename delete/remove to close

Reply:
The rename is due to the copy update from the designs: https://www.figma.com/file/7KIYbhoqNGAIFonE0w9TDz/Messages-for-Mobile?type=design&node-id=1447-349773&mode=design&t=OtI9WtlvD7M56ztS-0

Reply:
yeah, but it doesn't mean we should rename it in the code base, copy is enough, its still should have same functionality, or?

Reply:
The functionality changed, here's a link to the discuss:

https://discord.com/channels/624634427930312714/928625369542713396/1167063940816179271